packages/server/coretest/__tests__/StitchAppClientIntTests.ts

```diff
@@ -518,6 +522,36 @@ describe("StitchAppClient", () => {
     expect(client.auth.listUsers().length).toBe(2);
   });
 
+  it("should authenticate using a custom function", async () => {
```

Comment:
can we also add some fail tests as well as tests that actually use whats in the payload?